public/data/sarahlean.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "Sarah lean",
+  "type": "personal",
+  "bio": "Community Evangelist",
+  "avatar": "https://github.com/weeyin83.png",
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/weeyin83",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/techielsas",
+      "icon": "twitter"
+    },
+    {
+      "name": "Learn more about Microsoft Azure on my YouTube channel",
+      "url": "https://youtube.com/techielass",
+      "icon": "youtube"
+    },
+    {
+      "name": "Follow me on LinkedIn",
```

Comment:
```suggestion
      "name": "Connect with me on LinkedIn",
```

Reply:
This is personal choice of the author 😅 Please mark this as resolved.

Reply:
Thanks for the suggestion @Panquesito7 but as @krishguptadev says, this is my personal choice.